[WIP] Support C++ tensor advanced indexing

yf225 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#28524 [WIP] Support C++ tensor advanced indexing**
* #28523 [BC-breaking] Fix bugs in torch::tensor constructor

Differential Revision: [D18234626](https://our.internmc.facebook.com/intern/diff/D18234626)

Commit:
Support C++ tensor advanced indexing

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"




[ghstack-poisoned]

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

kostmo commented:
## CircleCI build failures summary
As of commit fabc7b2:
* 0/1 flaky



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=fabc7b22bde2f65916bdea0c2c9329309e7edfb4).

---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head

Commit:
Update on "[WIP] Support C++ tensor advanced indexing"

Support C++ tensor advanced indexing

gh-metadata: pytorch pytorch 28524 gh/yf225/17/head
